VehicleMsg cur_vehicle_state
CurrentLaneMsg current_lane
PerpendicularLaneMsg[] next_intersection
ParallelLaneMsg[] adjacent_lanes
float64 speed_limit
RewardInfoMsg reward_info
GlobalPathMsg global_path
GlobalPathMsg intersection_global_path